### PR TITLE
Proposed minor improvements to chapter 1

### PR DESCRIPTION
**Pull request by tkoolen**
* be a little more consistent/careful in the usage of dynamical system vs. control system
* improve nonholonomic constraints section a bit
* spelling
* minor other tweaks

### CODE REVIEW

File: underactuated.html [hunk 1]
```diff
@@ -788,7 +803,7 @@
 
 <li> Legged robots are underactuated.  Consider a legged machine with $N$
 internal joints and $N$ actuators.  If the robot is not bolted to the ground,
-then the degrees of freedom of the system include both the internal joints and
+then the degrees of freedom (DOFs) of the system include both the internal joints and
```
Comment (tkoolen) [inline underactuated.html]: One thing I noted is that we never properly define degree of freedom.

File: underactuated.html [hunk 2]
```diff
@@ -491,34 +491,41 @@
...
+$(\bq,\dot{\bq})$ at time $t$ if it is not able to command an instantaneous acceleration
 in an arbitrary direction in $\bq$: \begin{equation} \textrm{rank}\left[{\bf
 f}_2(\bq,\dot{\bq},t)\right] < \dim\left[\bq\right].
 \label{eq:underactuated_def}\end{equation} </div>
 
-<p>Notice that whether or not a control system is underactuated may depend on
-the state of the system, although for most systems (including all of the systems
+<p>Notice that whether or not a dynamical system is underactuated may depend on
+the state of the system and on time explicitly, although for most systems (including all of the systems
 in this book) underactuation is a global property of the system.</p>
```
Comment (tkoolen) [inline underactuated.html]: Should we add a note stating that in general a system is underactuated if it is underactuated for *any* state or time?

File: underactuated.html [hunk 3]
```diff
@@ -739,37 +748,43 @@
...
-underactauted but would not typically be called a nonholonmic system.  However,
+necessarily true&mdash;the double pendulum system which is missing an actuator is
+underactuated but would not typically be called a nonholonomic system.  However,
 while the definition of a nonholonomic constraint is clear, in my view the
 definition used in the literature for a "nonholonomic system" is more
 ambiguous&dagger;<sidenote>&dagger; Why, for instance, are the Lagrangian
-equations of motion for the double pendulum--clearly non-integrable-- not
+equations of motion for the double pendulum&mdash;clearly non-integrable&mdash;not
 considered nonholonomic constraints?</sidenote>, and we'll generally avoid using
 the term. </p>
```
Comment (tkoolen) [inline underactuated.html]: My interpretation: Lagrangian equations of motion are a constraint of the form phi(q, qd, qdd, t) = 0 and cannot be written in the form phi(q, qd, t) = 0 required for a nonholonomic constraint.

**Comment by tkoolen** [comment]
Any comments on these changes?

File: underactuated.html (hunk 1 above)
Comment (RussTedrake) [inline underactuated.html]: true.  but we never defined "robot" or "joints" either.  I think it's clear enough.

File: underactuated.html (hunk 2 above)
Comment (RussTedrake) [inline underactuated.html]: yes, that's reasonable.

File: underactuated.html [hunk 4]
```diff
@@ -622,20 +631,19 @@
 full row rank, is dim$[\bu] < $ dim$[\bq]$.  This is the case when a robot has
 joints with no motors. But this is not the only case.  The human body, for
 instance, has an incredible number of actuators (muscles), and in many cases has
-multiple muscles per joint; despite having more actuators that position
+multiple muscles per joint; despite having more actuators than position
 variables, when I jump through the air, there is no combination of muscle inputs
 that can change the ballistic trajectory of my center of mass (barring
-aerodynamic effects).  My control system is underactuated.</p>
+aerodynamic effects). I am underactuated.</p>
```
Comment (RussTedrake) [inline underactuated.html]: i prefer to talk about the control system being underactuated (and don't want to change control system to dynamical system throughout).  There are some cases where the same mecnhanical system may be underactuated or not depending on which control inputs are being used.

File: underactuated.html [hunk 5]
```diff
@@ -739,37 +748,43 @@
...
-underactuation.  Briefly, a nonholonomic constraint is a constraint on the
-derivatives of the system which cannot be integrated into a constraint on the
-positions of the system.  An automobile or traditional wheeled robot provides a
+underactuation.  Briefly, a nonholonomic constraint is a constraint
+of the form $\phi({\bf q}, {\bf \dot{q}}, t) = 0$, which cannot be
+integrated into a constraint of the form $\phi({\bf q}, t) = 0$ (a holonomic constraint).
+A nonholonomic constraint does not restrain the possible configurations of
+the system, but rather the manner in which those configurations can be reached.
+While a holonomic constraint reduces the number of degrees of freedom of a system by one,
+a nonholonomic constraint does not. An automobile or traditional wheeled robot provides a
```
Comment (RussTedrake) [inline underactuated.html]: that's better.  thanks.

File: underactuated.html (hunk 3 above)
Comment (RussTedrake) [inline underactuated.html]: nice.  i guess i agree.

**Comment by RussTedrake** [comment]
very nice, much appreciated.  i had already made some of these (the typo corrections) on my read through, and didn't agree with them all.  so i've made the changes locally, but will credit you on the commit message.

**Comment by tkoolen** [comment]
Great, thanks!

**Pull request closed without merging**